package.json

```diff
@@ -220,7 +220,8 @@
     "@parcel/rust": "2.12.1-dev.3224",
     "@parcel/types": "2.0.0-dev.1601",
     "@parcel/utils": "2.0.0-dev.1601",
-    "@parcel/workers": "2.0.0-dev.1601"
+    "@parcel/workers": "2.0.0-dev.1601",
+    "aria-query": "5.3.0"
```

Comment:
have to resolve this so that we don't special case based on react version in the tests I had to fix

Reply:
whelp, it didn't work
will work with @LFDanLu to see how we want to try to handle this

Reply:
After chatting, we've decided to special case in the few tests which are affected